File: app/src/main/java/com/ch/yoon/kakao/pay/imagesearch/repository/model/imagesearch/response/SearchMetaInfo.java
package com.ch.yoon.kakao.pay.imagesearch.repository.model.imagesearch.response;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

/**
 * Creator : ch-yoon
 * Date : 2019-08-01.
 */
public final class SearchMetaInfo {

    @SerializedName("is_end")
    private final boolean isEnd;

    public SearchMetaInfo(boolean isEnd) {
        this.isEnd = isEnd;
    }

    public boolean isEnd() {
        return isEnd;
    }

    @NonNull
    @Override
    public String toString() {
        return "SearchMetaInfo{, isEnd=" + isEnd + '}';
    }

}

File: app/src/main/java/com/ch/yoon/kakao/pay/imagesearch/data/remote/kakao/response/SearchMetaInfo.kt
package com.ch.yoon.kakao.pay.imagesearch.data.remote.kakao.response

import com.google.gson.annotations.SerializedName

/**
 * Creator : ch-yoon
 * Date : 2019-10-29.
 */
data class SearchMetaInfo(
    @SerializedName("is_end") val isEnd: Boolean
)